rllib/utils/pre_checks.py

```diff
@@ -3,18 +3,17 @@
 import sys
 import traceback
 
-from ray.rllib.env import BaseEnv, MultiAgentEnv, RemoteVectorEnv, VectorEnv
 import gym
 
 logger = logging.getLogger(__name__)
 
-SUPPORTED_ENVIRONMENT_TYPES = (BaseEnv, gym.Env, MultiAgentEnv,
-                               RemoteVectorEnv, VectorEnv)
 
-
-def check_env(
-        env: [BaseEnv, gym.Env, MultiAgentEnv, RemoteVectorEnv,
-              VectorEnv]) -> None:
+def check_env(env: ["BaseEnv", "gym.Env", "MultiAgentEnv", "RemoteVectorEnv",
```

Comment:
Does anyone know if this will fail sphinx build? 

@sven1977.

I need to do this because I need to local import these packages otherwise I have a circular import issue.

Reply:
Can you use `Union["BaseEnv", ...]` instead?
Quotes are fine if you want to avoid circular imports.
What also may help is to use `from typing import TYPE_CHECKING` (search for this in rllib/ to see examples of how imports can be conditioned on whether we are currently type checking or not). But this may not always help.